lib/live_debugger_refactor/services/process_monitor/gen_servers/process_monitor.ex

```diff
@@ -0,0 +1,92 @@
+defmodule LiveDebuggerRefactor.Services.ProcessMonitor.GenServers.ProcessMonitor do
+  @moduledoc """
+  This module is monitoring the status of LiveView processes created by a debugged application.
+  """
+
+  use GenServer
+
+  alias LiveDebuggerRefactor.CommonTypes
+  alias LiveDebuggerRefactor.Bus
+  alias LiveDebuggerRefactor.Services.ProcessMonitor.Actions
+  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceCalled
+  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceReturned
+
+  import LiveDebuggerRefactor.Services.Helpers
+
+  @type state :: %{
+          pid() => MapSet.t(CommonTypes.cid())
+        }
+
+  def start_link(opts \\ []) do
+    GenServer.start_link(__MODULE__, opts, name: __MODULE__)
+  end
+
+  @impl true
+  def init(_opts) do
+    Bus.receive_traces!()
+
+    ok(%{})
+  end
+
+  @impl true
+  def handle_info(%TraceReturned{function: :render, cid: cid, context: %{pid: pid}}, state)
+      when is_map_key(state, pid) and not is_nil(cid) do
+    state
+    |> maybe_register_component_created(pid, cid)
+    |> noreply()
+  end
+
+  @impl true
+  def handle_info(%TraceReturned{function: :render, cid: nil, context: %{pid: pid}}, state)
+      when not is_map_key(state, pid) do
+    state
+    |> Actions.register_live_view_born(pid)
+    |> noreply()
+  end
+
+  @impl true
+  def handle_info(
+        %TraceCalled{
+          module: Phoenix.LiveView.Diff,
+          function: :delete_component,
+          cid: cid,
+          context: %{pid: pid}
```

Comment:
I'm wondering actually if `pid` in such case shouldn't be a part of actual event and context 🤔 Not sure if context is useful now when we decided to store more inside events... Maybe we could keep there info about broadcaster of this event, like its PID or broadcaster module